apps/meteor/ee/app/livechat-enterprise/server/api/units.ts

```diff
@@ -46,7 +46,7 @@ API.v1.addRoute(
 		},
 		async post() {
 			const { unitData, unitMonitors, unitDepartments } = this.bodyParams;
-			return LivechatEnterprise.saveUnit(null, unitData, unitMonitors, unitDepartments);
+			return API.v1.success(LivechatEnterprise.saveUnit(null, unitData, unitMonitors, unitDepartments));
```

Comment:
Since we have a permission check in this endpoint, so can we remove it from the method to avoid re-checking?